config/autostart.go

```diff
@@ -38,33 +39,56 @@ func getLaunchdAgentPath() *paths.Path {
 func InstallPlistFile() {
 	launchdAgentPath := getLaunchdAgentPath()
 	if !launchdAgentPath.Exist() {
-		err := writePlistFile(launchdAgentPath)
+		writeLoadExit(launchdAgentPath)
+	} else {
+		// we already have an existing launchd plist file, so we check if it's updated
+		launchAgentContent, _ := launchdAgentPath.ReadFile()
+		launchAgentContentNew, _ := getLaunchdAgentDefinition()
+		if bytes.Equal(launchAgentContent, launchAgentContentNew) {
+			log.Infof("the autostart file %s already exists: nothing to do", launchdAgentPath)
+		} else {
+			log.Infof("the autostart file %s needs to be updated", launchdAgentPath)
+			removePlistFile()
+			writeLoadExit(launchdAgentPath)
```

Comment:
In this case, `launchd` will not auto-load the app. I tried also to `launchctl unload` and `launchctl load` manually but no autostart of the app either. The effect is that you must run the app two times to get it start:
1. the first run will update the plsit file and exit
2. the second run will actually execute the app

Reply:
Ok, I will remove the `os.Exit(0)` in case the plist needs to be updated. This way should work

Reply:
should be solved by 0efac75